Makefile

```diff
@@ -1,8 +1,13 @@
 # Set to 'yes' to include debugging information, e.g. DEBUG=yes make -e
-DEBUG		:= no
+DEBUG		:= yes
```

Comment:
Sorry, that's a change I made in order to build the debug rpm and I forgot switch to no again. Please ignore it.